sphinx/util/inspect.py

```diff
@@ -108,6 +108,10 @@ def safe_getattr(obj, name, *defargs):
     try:
         return getattr(obj, name, *defargs)
     except Exception:
+        # for NotImplementedError mixed abstractproperty, let's check for the
+        # property name before raising an error
```

Comment:
How about "try to read attributes directly for properties method which raises exceptions"?
(Please arrange freely because I'm not good at English...)